test/_external/test262/skiplist

```diff
@@ -135,3 +135,6 @@ test/language/module-code/namespace/internals/own-property-keys-sort.js
 
 # NEEDS INVESTIGATION!
 test/language/module-code/namespace/internals/set.js
+
+# NEEDS INVESTIGATION! -> only fails on node v6.14.3
+test/language/module-code/namespace/internals/object-keys-binding-uninit.js
```

Comment:
This is just a V8 tweak. In older Node they didn't error. In newer ones they do. We support erroring whenever Node does so this can be gated to whatever Node it's supported in. Related to https://bugs.chromium.org/p/v8/issues/detail?id=7780.

Reply:
ah, ok. it seems to be on the v6 branch only I believe. I'll implement a condition for _allowed_ versions or something like that (e.g. node6, harmony, etc.)